PREREQS.md

```diff
@@ -28,6 +28,22 @@ sudo apt install mesa-vulkan-drivers mesa-vulkan-drivers:i386
 
 Provide your user password when requested and reboot after the last command completes to ensure the driver has updated correctly.
 
+On Radeon R9 200/300 series you have to blacklist radeon module and add following parameters to amdgpu:
+
+```
+echo "blacklist radeon" | sudo tee --append /etc/modprobe.d/blacklist.conf
```

Comment:
Might want to have a warning here. It's not enabled by default because it's not done even with latest kernel releases